pkg/tsdb/parca/query.go

```diff
@@ -123,54 +128,161 @@ type CustomMeta struct {
 // [level, value, label] columns and by ordering the items in a depth first traversal order we can recreate the whole
 // tree back.
 func responseToDataFrames(resp *connect.Response[v1alpha1.QueryResponse]) *data.Frame {
-	if flameResponse, ok := resp.Msg.Report.(*v1alpha1.QueryResponse_Flamegraph); ok {
-		frame := treeToNestedSetDataFrame(flameResponse.Flamegraph)
-		frame.Meta = &data.FrameMeta{PreferredVisualization: "flamegraph"}
-		return frame
+	if flameResponse, ok := resp.Msg.Report.(*v1alpha1.QueryResponse_FlamegraphArrow); ok {
+		return arrowToNestedSetDataFrame(flameResponse.FlamegraphArrow)
 	} else {
-		panic("unknown report type returned from query")
+		// TODO: Can we be nicer about signaling users to update to have the latest APIs?
+		panic("unknown report type returned from query. update parca?")
```

Comment:
Is there a way to support both? I assume in the future they can diverge so we should push people to use the new API but I would think for now, supporting both and logging some warning would be nicer to the users.

Reply:
We don't have a meta API to discover which APIs are supported. Maybe gRPC has something via reflection. Or we simply try the newer one and fall back to the older one if it is not available. 
In reality, I don't expect many people to encounter this. There are so many developments in Parca that most people quickly upgrade.

Reply:
Hmm I would not mind deferring to you on this but guess @ivanahuckova should have the final say as they are the owners of this now.

In case we go with just hard update, I guess it would make more sense to then instead of panicking return the error in the response. Before the panic was there because I didn't think that could reasonably happen but in this case it would be sort of expected sometimes.

Reply:
I agree with @aocenas, I would prefer if we supported both, but if you @metalmatze think that the effort does not make sense, then I would:
- Instead of panicking, returning Error repose that would mention min parca version that user should update to
- Adding supported parca version to the documentation (see example in Loki data source documentation: https://grafana.com/docs/grafana/latest/datasources/loki/#supported-loki-versions)
- Adding information that the supported version of parca was changed to changelog

Reply:
Added the note to the Parca datasource documentation. 
I'm not sure how to add this change to the changelog. Do I simply add a line to `CHANGELOG.md`? I'm not sure where exactly this would fit? Which version etc.

Reply:
Done ✅  - I've added  `add to changelog` label and updated the PR title that will be in the changelog.

Reply:
Updated. 
This somehow got lost during a rebase, I think.